shoop/core/order_creator/_creator.py

```diff
@@ -23,8 +23,25 @@
 
 class OrderCreator(object):
 
-    def __init__(self, request):
-        self.request = request  # TODO: Get rid of `request`?
+    def __init__(self, request=None, remote_address=None):
+        """
+        Initialize order creator.
+
+        :type request: django.http.HttpRequest|None
+        :param request:
+          Optional request object for backward compatibility.  Used only
+          for initializing remote address: If request is given and
+          `remote_address` is not, then `remote_address` will
+          initialized to ``request.META.get("REMOTE_ADDRESS")``.
+        :type remote_address: str
+        :param remote_address:
+          Remote IP address as string (IPv4 or IPv6).  If given, will be
+          stored to `~shoop.core.models.Order.ip_address` field of
+          created orders.
+        """
+        self.remote_address = (
```

Comment:
It's now saved to OrderSource.ip_address and OrderCreator shouldn't be created with a request anymore.